src/cli.rs

```diff
@@ -14,25 +14,29 @@ pub fn cli() -> Command {
         .arg_required_else_help(true)
         .after_help(indoc! {r#"List of supported codecs
 
-| Image Format  | Input | Output | Note        |
-| ------------- | ----- | ------ | ----------- |
-| avif          | O     | O      | Static only |
-| bmp           | O     | X      |             |
-| farbfeld      | O     | O      |             |
-| hdr           | O     | O      |             |
-| jpeg          | O     | O      |             |
-| jpeg-xl       | O     | O      |             |
-| mozjpeg(jpeg) | O     | O      |             |
-| png           | O     | O      | Static only |
-| ppm           | O     | O      |             |
-| psd           | O     | X      |             |
-| qoi           | O     | O      |             |
-| webp          | O     | O      | Static only |
-
-        List of supported preprocessing options
-
-        - Resize
-        - Quantization"#})
+
+| Image Format  | Input | Output | Note            |
+| ------------- | ----- | ------ | --------------- |
+| avif          | O     | O      | Static only     |
+| bmp           | O     | X      |                 |
+| farbfeld      | O     | O      |                 |
+| gif           | O     | X      | Static only     |
```

Comment:
Also moving from image crate to zune-image introduced regressions in input formats, gif, ico and tiff are not supported. I can add these as a separate task